:

```diff
@@ -0,0 +1,32 @@
+# .bashrc
```

Comment:
Dear Youenn, thx for your PR! Not sure, if this file should be part of it? Thank you and with kind regards, Michael

Reply:
Wow, no idea how this happened !